components/UniversalGlossary.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="universal-glossary__container">
+    <bx-data-table>
+      <bx-table>
+        <bx-table-head>
+          <bx-table-header-row class="universal-glossary__table-header">
+            <bx-table-header-cell>Notation</bx-table-header-cell>
+            <bx-table-header-cell>Description</bx-table-header-cell>
+          </bx-table-header-row>
+        </bx-table-head>
+        <bx-table-body>
+          <bx-table-row
+            v-for="{ notation, description } in Object.values(glossary)"
+            :key="notation"
+            class="universal-glossary__table-row"
+          >
+            <bx-table-cell class="universal-glossary__table-row__symbol">
+              <RendererMath :source="notation"></RendererMath>
+            </bx-table-cell>
+            <bx-table-cell class="universal-glossary__table-row__description">{{
```

Comment:
```suggestion
            <bx-table-cell class="universal-glossary__notation__description">{{
```
If you accept the previous comment